test/common/http/utility_test.cc

```diff
@@ -126,13 +126,23 @@ TEST(HttpUtility, parseHttp2Settings) {
 
 TEST(HttpUtility, getLastAddressFromXFF) {
   {
-    const std::string first_address = "34.0.0.1";
-    const std::string second_address = "10.0.0.1";
+    const std::string first_address = "192.0.2.10";
+    const std::string second_address = "192.0.2.1";
+    const std::string third_address = "10.0.0.1";
     TestHeaderMapImpl request_headers{
-        {"x-forwarded-for", fmt::format("{0}, {0}, {1}", first_address, second_address)}};
+        {"x-forwarded-for", first_address + ", " + second_address + ", " + third_address}};
```

Comment:
Sorry I actually meant remove the first_address, etc. variables and literally do:
```
{"x-forwarded-for", "129.0.2.10, 192.0.2.1, 10.0.0.1"}
```

:) But just a nit/preference if you feel otherwise.